packages/app/src/components/markdown.tsx

```diff
@@ -10,9 +10,10 @@ import { Link } from '~/utils/link';
 import { DisplayOnMatchingQueryCode } from './display-on-matching-query-code';
 import { Message } from './message';
 import { Anchor } from './typography';
-
+import { ElementType } from 'react';
 interface MarkdownProps {
   content: string;
+  renderersOptions?: { [nodeType: string]: ElementType };
```

Comment:
```suggestion
  rendererOverrides?: { [nodeType: string]: ElementType };
```
Since you're overriding renderers, I don't think the term `options` applies here.